src/main/java/edu/kit/provideq/toolbox/demonstrators/DemonstratorConfiguration.java

```diff
@@ -16,9 +19,14 @@ public class DemonstratorConfiguration {
   public static final ProblemType<String, String> DEMONSTRATOR = new ProblemType<>(
       "demonstrator",
       String.class,
-      String.class
+      String.class,
+      null
   );
 
+  private Function<String, Bound> demonstratorEstimator() {
```

Comment:
I don't think this is needed as the function is not used anyway. Passing null for the estimator should be enough.